package.json

```diff
@@ -38,7 +38,7 @@
     "eslint-plugin-react": "^7.19.0",
     "eslint-plugin-react-hooks": "^3.0.0",
     "husky": "^4.2.5",
-    "lerna": "^3.22.1",
+    "lerna": "^4.0.0",
```

Comment:
no breaking changes with this major bump i assue?

Reply:
Just a node version deprecation that doesn't concern us.